src/rngs/small.rs

```diff
@@ -83,9 +83,6 @@ impl SeedableRng for SmallRng {
 
     #[inline(always)]
     fn from_seed(seed: Self::Seed) -> Self {
-        // With MSRV >= 1.77: let seed = *seed.first_chunk().unwrap();
```

Comment:
I'm not sure what was the reason for these 3 lines. While they don't affect the PR directly, I thought I'd remove them as a tiny clean-up.

Reply:
The purpose of the code you deleted is, I think, compatibility with 32-bit platforms where `Rng` has a different seed size. The comment points out a neater implementation we could use in the future.